src/components/inputs/selection-plan-dropdown.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Dropdown } from "openstack-uicore-foundation/lib/components";
+
+const SelectionPlanDropdown = ({ id, value, onChange, selectionPlans, ...rest }) => {
```

Comment:
why we are using the entire selection plan data set ?
lets use a dynamic approach like here https://github.com/OpenStackweb/openstack-uicore-foundation/blob/main/src/components/inputs/ticket-types-input.js